### DISCUSSION TITLE
error installation

### DISCUSSION THREAD

**Discussion by vovabezpaleva** [wg-easy/wg-easy#974] [Q&A]
Docker  was completed successfully and wg-easy installation error

<img width="365" alt="ewet2" src="https://github.com/wg-easy/wg-easy/assets/166128572/6a6ddd3b-8094-4996-830c-147978692405">

**Comment by lewis-precision** [comment]
Hi there, try typing out the whole command first instead of copy-pasting as this can sometimes cause issue.

**Comment by vovabezpaleva** [comment]
> Hi there, try typing out the whole command first instead of copy-pasting as this can sometimes cause issue.

OK thanks

<img width="521" alt="sdfw32q3" src="https://github.com/wg-easy/wg-easy/assets/166128572/379dd9cf-6278-4977-bfe1-25414504ed99">


error again ;(

This is my first installation of Lunex and I'm learning and it's super interesting

**Comment by methbkts** [comment]
I advise you to use "Docker Compose" instead, it's easier to maintain in the long term.

Copy the [docker-compose.yml](https://github.com/wg-easy/wg-easy/blob/master/docker-compose.yml) file, make necessary adjustments, save the file and execute `docker compose up --detach`

**Comment by vovabezpaleva** [comment]
It worked. But the GUI is suddenly available publicly for everyone to access. How to close the GUI with a password?

My docker-compose.yml file:

`

      # Optional:
      # - PASSWORD=12345qwert
      # - WG_PORT=51820
      # - WG_DEFAULT_ADDRESS=10.8.0.x`

**Comment by eandersons** [comment]
Uncomment (remove the `# ` from) the line `- PASSWORD=12345qwert` and replace `12345qwert` with a strong password.

**Comment by vovabezpaleva** [comment]
ок! super password `12345qwert`  changed :))
